data-prepper-plugins/http-source-common/build.gradle

```diff
@@ -9,6 +9,7 @@ plugins {
 
 dependencies {
     implementation project(':data-prepper-plugins:common')
+    implementation project(':data-prepper-test-common')
```

Comment:
Please make this `testImplementation` and move toward the bottom to sit with the other `testImplementation` dependencies.